hack/util.sh

```diff
@@ -570,6 +570,11 @@ function cleanup_openshift {
 	delete_empty_logs
 	truncate_large_logs
 
+	if grep -q 'no Docker socket found' "${LOG_DIR}/openshift.log"; then 
+		# the Docker daemon crashed, we need the logs
+		journalctl --unit docker.service > "${LOG_DIR}/docker.log"
```

Comment:
and maybe gzip the log file so when people downloading it they don't have fetch xMB of logs ;-)